code/osapi/DebugWindow.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "globalincs/pstypes.h"
+#include "osapi/osapi.h"
+
+namespace osapi {
+
+class DebugWindow {
```

Comment:
Would you please doxy the rest of the class? Each method and the class itself should be documented, and so should any of the non-trivial members.